components/PhotoButton/PhotoButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Button } from "react-native";
+interface PhotoButtonProps {
+	onPress: VoidFunction;
+	color: string;
+}
+
+const PhotoButton = (props: PhotoButtonProps): JSX.Element => {
+	return (
+		<Button
+			onPress={props.onPress}
+			title=""
```

Comment:
nit: Would prefer to have a word or something to indicate that this is a button. Currently its just a white box at the bottom.

Reply:
Yeah it is pretty cryptic, but I left out text now because when it is styled there will not be any text inside, and it will be clearer what it does